spec/models/embed/purl_spec.rb

```diff
@@ -230,6 +230,14 @@
           expect(file.location).to eq 'http://stacks.stanford.edu/file/druid:abc123/Title_of_the_PDF.pdf'
         end
       end
+      describe 'video_data' do
+        before { stub_purl_response_with_fixture(multi_media_purl) }
+        let(:video) { Embed::PURL.new('JessieSaysNo.mp4').contents.first.files.first }
```

Comment:
d'oh, yeah, was thinking it took a file name for some reason when i was writing this.  will do the same as the other examples in this file.